lib/blocktree/blocktree.go

```diff
@@ -133,20 +133,20 @@ func (bt *BlockTree) getNode(h Hash) (ret *node) {
 // Prune sets the given hash as the new blocktree root,
 // removing all nodes that are not the new root node or its descendant
 // It returns an array of hashes that have been pruned
-func (bt *BlockTree) Prune(finalised Hash) (pruned []Hash) {
+func (bt *BlockTree) Prune(finalised Hash) []Hash {
```

Comment:
nit What's wrong with the named return? :smiling_face_with_tear:

Reply:
nothing, i was just trying things out lol, changed it back